tsconfig.json

```
@@ -36,7 +36,7 @@
   ],
   "include": [
     "src/**/*.ts",
-    "src/**/*.tsx"
+    "src/**/*.tsx",
```

this is invalid JSON
```suggestion
    "src/**/*.tsx"
```